go.mod

```diff
@@ -183,36 +183,20 @@ require (
 require (
 	github.com/PuerkitoBio/purell v1.1.1 // indirect
 	github.com/PuerkitoBio/urlesc v0.0.0-20170810143723-de5bf2ad4578 // indirect
-	github.com/ahmetb/gen-crd-api-reference-docs v0.3.0 // indirect
+	github.com/Yamashou/gqlgenc v0.0.8 // indirect
 	github.com/emicklei/go-restful v2.9.5+incompatible // indirect
-	github.com/fatih/color v1.13.0 // indirect
 	github.com/go-openapi/jsonpointer v0.19.5 // indirect
 	github.com/go-openapi/jsonreference v0.19.5 // indirect
 	github.com/go-openapi/swag v0.19.14 // indirect
-	github.com/gobuffalo/flect v0.2.3 // indirect
 	github.com/google/gnostic v0.5.7-v3refs // indirect
-	github.com/hashicorp/hcl v1.0.0 // indirect
-	github.com/inconshreveable/mousetrap v1.0.0 // indirect
 	github.com/josharian/intern v1.0.0 // indirect
-	github.com/magiconair/properties v1.8.5 // indirect
 	github.com/mailru/easyjson v0.7.6 // indirect
-	github.com/mattn/go-colorable v0.1.12 // indirect
 	github.com/munnerz/goautoneg v0.0.0-20191010083416-a7dc8b61c822 // indirect
-	github.com/pelletier/go-toml v1.9.4 // indirect
-	github.com/rs/zerolog v1.26.1 // indirect
-	github.com/russross/blackfriday/v2 v2.1.0 // indirect
-	github.com/spf13/afero v1.8.0 // indirect
-	github.com/spf13/cast v1.4.1 // indirect
-	github.com/spf13/cobra v1.3.0 // indirect
-	github.com/spf13/jwalterweatherman v1.1.0 // indirect
-	github.com/spf13/viper v1.10.1 // indirect
-	github.com/subosito/gotenv v1.2.0 // indirect
-	github.com/vektra/mockery/v2 v2.10.0 // indirect
-	k8s.io/apiextensions-apiserver v0.23.0 // indirect
-	k8s.io/gengo v0.0.0-20210813121822-485abfe95c7c // indirect
-	k8s.io/klog v1.0.0 // indirect
-	sigs.k8s.io/controller-tools v0.6.2 // indirect
-	sigs.k8s.io/kustomize/kyaml v0.10.17 // indirect
+	github.com/rivo/uniseg v0.2.0 // indirect
+	github.com/schollz/progressbar/v3 v3.8.6 // indirect
+	github.com/vektah/gqlparser/v2 v2.4.6 // indirect
 )
 
 replace k8s.io/klog/v2 => github.com/Raffo/knolog v0.0.0-20211016155154-e4d5e0cc970a
+
+replace github.com/Yamashou/gqlgenc => github.com/pluralsh/gqlgenc v0.0.9
```

Comment:
can you clarify this?

Reply:
we had to fork this library because it uses some custom JSON parser that throws nil pointer exception when trying to unmarshal our GraphQL schema:
https://github.com/Yamashou/gqlgenc/pull/141

We can try to contribute to this library and fix the issue but it's not simple and can take some time.

Reply:
@szuecs right now the problem is fixed in the `github.com/Yamashou/gqlgenc` library and this line is not needed anymore